src/rgw/rgw_op.h

```diff
@@ -183,6 +184,10 @@ class RGWOp : public DoutPrefixProvider {
   std::ostream& gen_prefix(std::ostream& out) const override;
   CephContext* get_cct() const override { return s->cct; }
   unsigned get_subsys() const override { return ceph_subsys_rgw; }
+
+  using client_id = rgw::dmclock::client_id;
+  virtual client_id dmclock_client() { return client_id::metadata; }
+  virtual rgw::dmclock::Cost dmclock_cost() { return 0; }
```

Comment:
is the dmclock_cost supposed to be implemented by the SubOp classes? A 0-cost would eventually hit the assert in RequestTag calculation as we assert everywhere that cost is +ve

Reply:
oh yeah, that assert was added in https://github.com/ceph/dmclock/pull/46, so the `return 0;` here should default to 1 instead